python/environment.yml

```
@@ -8,6 +8,15 @@ dependencies:
   # Code formatter and stuff
   - jedi-language-server
 
+  # Compilers
+  - libgcc-ng
+  - gcc_linux-64
+  - gxx_linux-64
```

Get `gcc=13`. Those `libgcc-ng` and `gcc_linux-64` packages should be pulled in automatically when installing `rust` from conda-forge, so no need to declare them explicitly.

```suggestion
  # Compilers
  - gcc=13
```

Note that typically we would set a dependency on [`c-compiler`](https://anaconda.org/conda-forge/c-compiler/files), but this has a pin to `gcc=12` currently, and we don't want that.